`4.17.0 Relevant` Deprecated NEO (#1405)
SAP/cloud-sdk: docs-java/faq.mdx

### Do You Support SAP Business Technology Platform - Neo?

:::warning warning

> We are planning to discontinue support of NEO with SAP Cloud SDK v5 by end of 2023.

Reviewer: ```suggestion
All the NEO specific classes and functions have been deprecated.
We are planning to discontinue support of NEO with SAP Cloud SDK v5 by end of 2023.
```

### Do You Support SAP Business Technology Platform - Neo?

> We are planning to discontinue support of NEO with SAP Cloud SDK v5 by end of the year.

Reviewer 2: Who knows when somebody is going to stumble over this sentence ... 😉

```suggestion
We are planning to discontinue support of NEO with SAP Cloud SDK v5 by end of 2023.
```

Reviewer 2: Also, to make this stick out even more, why not wrap it inside a `:::warn` box and leave the rest of this paragraph untouched.

> Therefore, we're announcing module deprecation in advance.

Reviewer 2: Personally, I don't think this sentence belongs here.

```suggestion
```

### Do You Support SAP Business Technology Platform - Neo?

:::warning warning
We are planning to discontinue support of NEO with SAP Cloud SDK v5 by end of 2023.
The Cloud Foundry environment is better suited for apps and SAP S/4HANA extension development.

> :::
> We still provide full support for SAP Business Technology Platform Neo.

Reviewer 2: Let's add this back in:

```suggestion
:::

We still provide full support for SAP Business Technology Platform Neo.
However, we do not recommend starting new projects with SAP BTP for the Neo environment.
```

### Do You Support SAP Business Technology Platform - Neo?

:::warning warning
We are planning to discontinue support of NEO with SAP Cloud SDK v5 by end of 2023.
The Cloud Foundry environment is better suited for apps and SAP S/4HANA extension development.
:::

We still provide full support for SAP Business Technology Platform Neo.
However, we do not recommend starting new projects with SAP BTP for the Neo environment.

> The Cloud Foundry environment is better suited for apps and SAP S/4HANA extension development.

Reviewer 2: Why are we keep removing sentences? 😅
Just add the box on top of this and we are good IMO.
